src/index.js

```diff
@@ -0,0 +1,128 @@
+// import axios from 'axios';
+'use strict';
+
+const state = {
+  temp: 32,
+};
+
+const plusClickCount = () => {
+  const plusContainer = document.getElementById('Temperature');
+  state.temp += 1;
+  plusContainer.textContent = state.temp;
+  changeColor();
+};
+
+const minusClickCount = () => {
+  const minusContainer = document.getElementById('Temperature');
+  state.temp -= 1;
+  minusContainer.textContent = state.temp;
+  changeColor();
+};
+//making text color different
+const changeColor = () => {
+  const tempColor = document.querySelector('#Temperature');
+  const newLS = document.getElementById('newLandscape');
+
+  if (state.temp < 49) {
+    tempColor.style.color = 'teal';
+    newLS.src = `assets/cold.jpeg`;
+  } else if (state.temp >= 50 && state.temp < 59) {
+    tempColor.style.color = 'green';
+    newLS.src = `assets/cool.jpeg`;
+  } else if (state.temp >= 60 && state.temp < 69) {
+    tempColor.style.color = 'yellow';
+    newLS.src = `assets/warm.jpeg`;
+  } else if (state.temp >= 70 && state.temp < 79) {
+    tempColor.style.color = 'orange';
+    newLS.src = `assets/beach.jpeg`;
+  } else if (state.temp >= 80) {
+    tempColor.style.color = 'red';
+    newLS.src = `assets/hot1.jpeg`;
+  }
```

Comment:
The logic here is correct, the only thing that is that you didn't need to put `state.temp >= 50 && state.temp < 59` and then `(state.temp >= 60 && state.temp < 69)`. You could just do `state.temp > 50` and `state.temp > 60`. These take care of the range between these two numbers.